setup_custom_mcp.py

```
@@ -0,0 +1,130 @@
+
+        response = (
+            input("Do you want to create backups and overwrite these files? (y/N): ")
+            .strip()
+            .lower()
+        )
+        if response not in ["y", "yes"]:
+            print("❌ Setup cancelled. Use --force to overwrite without prompting.")
+            return False
+
+        # Create backups
+        for file_path, description in existing_files:
```

Ruff (0.13.1)

Loop control variable `description` not used within loop body

(B007)